smartsim/_core/_install/configs/mlpackages/LinuxX64CUDA125.json

```
@@ -0,0 +1,64 @@
+{
+    "platform": {
+        "operating_system":"linux",
+        "architecture":"x86_64",
+        "device":"cuda-12.5"
+    },
+    "ml_packages": [
+        {
+            "name": "dlpack",
+            "version": "v0.5_RAI",
+            "pip_index": "",
+            "python_packages": [],
+            "lib_source": "https://github.com/RedisAI/dlpack.git"
+        },
+        {
+            "name": "libtorch",
+            "version": "2.4.0",
+            "pip_index": "https://download.pytorch.org/whl/cu124",
+            "python_packages": [
+                "torch==2.4.0+cu125",
+                "torchvision==0.19.0+cu125",
+                "torchaudio==2.4.0+cu125"
```

Sanity check: should these be
```suggestion
                "torch==2.4.0+cu124",
                "torchvision==0.19.0+cu124",
                "torchaudio==2.4.0+cu124"
```
? I don't think a `+cu125` version of these packages exist.